package-lock.json

```diff
@@ -13,9 +13,9 @@
       },
```

Comment:
Was this just an `npm upgrade`?

Reply:
I went through them all, checking latest versions. Then discovered the really quick way!